.github/README.md

```diff
@@ -1,17 +1,34 @@
+    <img src="https://img.shields.io/badge/Files-Website-F9B81F" alt="Files Website" /></a>
+  <a style="text-decoration:none" href="https://github.com/files-community/Files/actions/workflows/ci.yml">
+    <img src="https://github.com/files-community/Files/actions/workflows/ci.yml/badge.svg" alt="Files CI Status" /></a>
+  <a style="text-decoration:none" href="https://crowdin.com/project/files-app">
+    <img src="https://badges.crowdin.net/files-app/localized.svg" alt="Files Localization Status" /></a>
+  <a style="text-decoration:none" href="https://discord.gg/files">
+    <img src="https://img.shields.io/discord/725513575971684472?label=Discord&color=7289da" alt="Files Discord" /></a>
+</p>
+
+Files is a modern file manager that helps users organize their files and folders. Our mission with Files is to build the best file manager for Windows, and we’re proud to be building it out in the open so everyone can participate. User feedback helps shape the features we work on, & the bug reports on GitHub help to make Files more reliable. Built and maintained by the open-source community, Files features robust multitasking experiences, file tags, deep integrations, and an intuitive design.
+
+## Install Files
+
+Files is a community-driven project that depends on your support to grow and improve. Please consider purchasing Files through the Microsoft Store or supporting us on GitHub if you use the classic installer.
+
+You can also use the preview version alongside the stable release to get early access to new features and improvements.
+
+<p align="left">
+  <a style="text-decoration:none" href="ms-windows-store://pdp/?productid=9NGHP3DX8HDX">
+    <img src="../assets/StoreBadge.png" alt="Files @ Microsoft Store" width="220" /></a>
+</p>
 
-Introducing Files, the ultimate file manager app for Windows. With its sleek and intuitive design, navigating through your files has never been easier. Files features tabs for easy switching between different folders, a column view for quick file browsing, and dual pane support for efficient file management. In addition, you can easily create and extract archives with just a few clicks, making file compression and decompression a breeze.
+### Additional installation methods
 
-Files also offers advanced features such as file tagging for easy organization, support for QuickLook for previewing files without opening them, and the ability to customize the background color to match your personal style. Whether you're a power user or just looking for a better way to manage your files, Files has everything you need to keep your files organized and easily accessible. With its combination of powerful features and ease of use, Files is the ultimate file management solution for Windows.
+In addition to the [Classic installer](https://cdn.files.community/files/download/Files.Stable.exe), you can find additional installation methods on our [documentation](https://files.community/docs/install) site. 
```

Comment:
@0x5bfa can you create a badge for the classic installer? Doing this will allow us to display both download options side by side.